packages/nc-gui/composables/useLoadingIndicator/index.ts

```diff
@@ -0,0 +1,72 @@
+import { computed, ref } from '#imports'
```

Comment:
I didn't mean to check this file in but w/e, I was going to commit it at some point anyway 😄